### PR TITLE
Update to use new metadata spec

### LINKED ISSUE #224: Remove urth, support v1 metadata spec
Wait for https://github.com/jupyter-incubator/dashboards/issues/243 to get implemented, at least in a branch. Then:

* [ ] Remove all urth references in CSS classes and whatnot
* [ ] Support rendering of v1 metadata spec notebooks (https://github.com/jupyter-incubator/dashboards/wiki/Dashboard-Metadata-and-Rendering is finalized)
* [ ] Decide if/how we continue to support rendering of v0 spec notebooks

DO NOT add any new user-facing features (e.g., NO multi-layout support). This is a cleanup and switcheroo only.

### PR DESCRIPTION
**Pull request by jhpedemonte**
Also update example/test notebooks

Fixes #224 

cc @dalogsdon

### CODE REVIEW

File: app/handlebars-helpers.js [hunk 1]
```diff
@@ -4,29 +4,22 @@
...
-            return {
-                layout: 'grid',
-                cellMargin: config.get('DB_CELL_MARGIN'),
-                defaultCellHeight: config.get('DB_DEFAULT_CELL_HEIGHT'),
-                maxColumns: config.get('DB_MAX_COLUMNS')
-            };
-        } else {
-            return {};
-        }
-    },
```
Comment (jhpedemonte) [inline app/handlebars-helpers.js]: @dalogsdon: Do we need to have defaults for these values? Or will all created dashboards have it? What about notebooks with no layout?

If we don't need these any more, then we can remove from **config.json**, right?

Comment (dalogsdon) [inline app/handlebars-helpers.js]: The dashboard server will generate these in every notebook. The question is do we want to provide defaults if they are missing for any reason (like if a user deleted them) or always require them in the notebook file? If we always require them, we can remove defaults from `config.json` here.

File: routes/renderers.js [hunk 2]
```diff
@@ -88,25 +89,20 @@ function _renderDashboard(req, res, next, opts) {
...
             // get dashboard layout from notebook-level dashboard metadata
-            var dashboardLayout = (notebook.metadata &&
-                                  notebook.metadata.urth &&
-                                  notebook.metadata.urth.dashboard &&
-                                  notebook.metadata.urth.dashboard.layout) ||
-                                  NO_LAYOUT;
+            var activeView = getObject(notebook, 'metadata.extensions.jupyter_dashboards.activeView') ||
+                             NO_LAYOUT;
 
             // backwards compatibility - old grid layouts don't have layout set
```
Comment (dalogsdon) [inline routes/renderers.js]: Are we providing backwards compatibility in dashboards server?

cc @parente

**Comment by dalogsdon** [comment]
It looks like some notebook checkpoint files were checked in. See 2 inline comments above. Other than that, works for me and looks good.

File: routes/renderers.js (hunk 2 above)
Comment (jhpedemonte) [inline routes/renderers.js]: We can delete this backwards-compat layer.

File: app/handlebars-helpers.js (hunk 1 above)
Comment (jhpedemonte) [inline app/handlebars-helpers.js]: I think bulletproofing in this way should be handled in a separate issue. Right now, the code assumes that the dashboard was correctly generated by the dashboard extension.

**Comment by jhpedemonte** [comment]
Updated based on comments. Please re-review when you get a chance, @dalogsdon.

**Pull request merged**